examples/pxScene2d/src/Makefile

```diff
@@ -414,6 +419,10 @@ PXSCENE_EXTRA_DEFINES=-DPNG_APNG_SUPPORTED
 endif
 endif
 
+ifeq ($(HAVE_RT_REMOTE),1)
+EXTRA_COMPILER_FLAGS:=$(EXTRA_COMPILER_FLAGS) -DUSE_RT_REMOTE 
+LDFLAGS_FULL:=$(LDFLAGS_FULL) -L../../../remote -lrtRemote
```

Comment:
This will work when building the static version of the libs (libpxscene.a), but it won't be included when building the shared library version (libpxscene.so).  We build the shared libraries when developing to allow for faster work environments.  Can you please update to ensure librtRemote is linked when building the shared library version as well?